packages/edit-site/src/components/add-new-template/style.scss

```diff
@@ -12,33 +12,45 @@
 
 .edit-site-custom-template-modal {
 	&__contents {
-		> div {
-			text-align: center;
-			cursor: pointer;
+		> .components-button {
```

Comment:
Overriding styles of a component from the `@wordpress/components` package this way is not ideal:

- we _should not_ use hardcoded `@wordpress/components` classnames, as they are intended to be implementation details and they are not part of the components' public APIs. Instead, we should assign a `className` to the component we'd like to target (in this case, the `FlexItem`), and use that new classname as the CSS selector
- more in general, the amount of style overrides that are being applied to this button is worrying me a little. We should _really_ try to limit those to the minimum. If we find that our components can't fulfil our designs, we should discuss this mismatch and either update the designs, or update the components. Style overrides, such this specific one, can be a big obstacle to maintaining and improving the components package

I know that most of these issues existed before this PR, but it's this PR that brought this code snippet to my attention